advanced-integration/package.json

```diff
@@ -15,9 +15,11 @@
     "dotenv": "^16.0.0",
     "ejs": "^3.1.6",
     "express": "^4.17.3",
-    "node-fetch": "^3.2.1"
+    "node-fetch": "^3.2.1",
+    "kill-port": "^2.0.1",
```

Comment:
we don't use `kill-port` anywhere in the code. 
We only use that in codespaces configuration, we can even just add a flag in configuration to install that `npx kill-port --y`

Reply:
Reverted back and adjusted in devcontainer for same